docs/logging-usage.rst

```diff
@@ -206,11 +206,8 @@ Delete a metric:
    False
 
 
-Export log entries using sinks
-------------------------------
-
-Sinks allow exporting entries which match a given filter to Cloud Storage
-buckets, BigQuery datasets, or Cloud Pub/Sub topics.
+Export to Cloud storage
+=======================
```

Comment:
ISTM that these three sections should be sub-heads under `Export log entries using sinks`.